pkg/backup/backup.go

```diff
@@ -205,15 +235,34 @@ type tarWriter interface {
 
 // backupGroup backs up a single API group.
 func (kb *kubernetesBackupper) backupGroup(ctx *backupContext, group *metav1.APIResourceList) error {
-	var errs []error
+	var (
+		errs []error
+		pv   *metav1.APIResource
+	)
 
-	for _, resource := range group.APIResources {
+	processResource := func(resource metav1.APIResource) {
 		ctx.infof("Processing resource %s/%s", group.GroupVersion, resource.Name)
 		if err := kb.backupResource(ctx, group, resource); err != nil {
 			errs = append(errs, err)
 		}
 	}
 
+	for _, resource := range group.APIResources {
+		// do PVs last because if we're also backing up PVCs, we want to backup
```

Comment:
This works for now, but do you think longer-term we should have `prioritizedResources` for backups too?

Reply:
I'm actually going to want to make sure we have this order: pods, pvcs, pvs (anything else can come whenever). Maybe I'll do prioritized resources as part of my hooks PR. We don't necessarily have to expose it initially, but I think something a bit more flexible than a bool or two is going to be needed.

Reply:
Yeah, thought the same thing - if you're okay with it, I'll leave this as-is and either you or I can do priorities in a separate PR

Reply:
Fine as-is